emcc.py

```diff
@@ -1909,6 +1909,9 @@ def get_full_import_name(name):
          shared.Settings.MAXIMUM_MEMORY > 2 * 1024 * 1024 * 1024)):
       shared.Settings.CAN_ADDRESS_2GB = 1
 
+    if shared.Settings.EXCEPTION_HANDLING and shared.Settings.OPT_LEVEL >= 2 and not compile_only:
+      exit_with_error('wasm exception handling support is still experimental, and not supported in -O2+ yet')
```

Comment:
I think the warning should make the distinction between O2 at compile time and link time, since if I wanted to test a codebase with the best possible performance, then O3 at compile time and O1 at link time would come fairly close.

Reply:
Good point, edited to mentioning linking.